src/lib.rs

```diff
@@ -266,12 +266,19 @@ impl StreamDeck {
                 states.extend(vec![0u8; (self.kind.key_count() + 1) as usize]);
 
                 if data[9] != 0 {
-                    let key = if self.kind == Kind::Akp815 {
-                        inverse_key_index(&self.kind, data[9] - 1)
-                    } else {
-                        ajazz153_to_elgato_input(&self.kind, data[9] - 1)
+                    let key = match self.kind {
+                        Kind::Akp815 => inverse_key_index(&self.kind, data[9] - 1),
+                        Kind::Akp153 | Kind::Akp153E | Kind::Akp153R | Kind::MiraBoxHSV293S => ajazz153_to_elgato_input(&self.kind, data[9] - 1),
+                        Kind::MiraBoxDK0108D => data[9] - 1,
+                        _ => data[9] - 1,
                     };
 
+                    // This device can slide its view to show keys 48, 49 and 50.
```

Comment:
@4ndv corrected me here about how the device works and I edited my original comment. So please could you use the edited version of my suggestion